quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/dist/approvals/cli/help/HelpCommandDistTest.testStartDevHelp.unix.approved.txt

```diff
@@ -150,14 +150,17 @@ HTTP(S):
 --https-protocols <protocols>
                      The list of protocols to explicitly enable. Default: TLSv1.3,TLSv1.2.
 --https-trust-store-file <file>
-                     DEPRECATED: The trust store which holds the certificate information of the
-                       certificates to trust.
+                     DEPRECATED. Use the System Truststore instead, see the docs for details. --
+                       The trust store which holds the certificate information of the certificates
+                       to trust.
 --https-trust-store-password <password>
-                     DEPRECATED: The password of the trust store file.
```

Comment:
I think the option description is a bit confusing. Perhaps:

```
DEPRECATED: The password of the trust store file.  Use the System Truststore instead, see the docs for details..
```

Reply:
Rewrote the message a bit.